pyaccel/lattice.py

```diff
@@ -528,17 +512,15 @@ def get_error_rotation_roll(lattice, indices):
     """
 
     #  processes arguments
-    indices, *_ = _process_args_errors(indices, 0.0)
+    indices, _, isflat = _process_args_errors(indices, 0.0)
 
     # loops over elements and gets error from R_IN
     values = []
     for segs in indices:
         for idx in segs:
             angle = _math.asin(lattice[idx].r_in[0, 2])
             values.append(angle)
-    if len(values) == 1:
-        return values[0]
-    return values
+    return _process_output(values, isflat)
```

Comment:
There's a bug here. Remember that values must have the same structure as indices. It must be a list of lists.

Reply:
fixed